qml/pages/SearchPage.qml

```diff
@@ -302,11 +302,11 @@ Dialog {
                 visible: searchType === "city"
 
                 menu: ContextMenu {
-                    MenuItem { text: "name" }
-                    MenuItem { text: "popularity" }
-                    MenuItem { text: "price" }
-                    MenuItem { text: "rating" }
-                    MenuItem { text: "stars" }
+                    MenuItem { text: qsTr("name") }
```

Comment:
Text from menu item is used in a request. I'll make mapping for request values in future commits.